src/pages/safety-&-side-effects.astro

```diff
@@ -1,18 +1,244 @@
+            <h4 class="text--center">Have heart problems</h4>
+            
+            <img src="/assets/images/safety-and-side-effects/icons/eye-icon.png" alt="">
+            <h4 class="text--center">Have eye problems</h4>
+            
+            
+            <img src="/assets/images/safety-and-side-effects/icons/pregnant-icon.png" alt="" srcset="">
+            
+            <div class="side-effects-page--problems-section__long">
+              <h4 class="text--center">Are pregnant or plan to become pregnant. Koselugo can harm an unborn baby</h4>
+
+              <ul class="side-effects-page--list">
+                <li>Your child’s healthcare provider should check to see if your child is pregnant before they begin treatment with Koselugo</li>
+                <li>Females who are able to become pregnant should use effective birth control (contraception) during treatment with Koselugo and for 1 week after the last dose</li>
+                <li>Males with female partners who are able to become pregnant should use effective birth control (contraception) during treatment with Koselugo and for 1 week after the last dose</li>
+                <li>Tell your child’s healthcare provider right away if your child becomes pregnant or thinks she may be pregnant during treatment with Koselugo</li>
+              </ul>
+
+            </div>
+
+            <img src="/assets/images/safety-and-side-effects/icons/breastfeed-icon.png" alt="" srcset="">
+
+            <div class="side-effects-page--problems-section__long">
+              
+              <h4 class="text--center">Are breastfeeding or plan to breastfeed</h4>
+
+              <ul class="side-effects-page--list">
+                <li>It is not known if Koselugo passes into breast milk</li>
+                <li>Your child should not breastfeed during treatment with Koselugo and for 1 week after the last dose</li>
+                <li>Talk to your child’s healthcare provider about the best way for your child to feed her baby during this time</li>
+              </ul>
+
+            </div>
+
+          </div>
+
+        </div>
+
+      </Section>
+
+      <section class="side-effects-page--secondary-section">
+        <p class="side-effects-page--disclaimer text--center color--aqua">
+          Tell your child’s healthcare provider about all the medicines your child takes, including prescription and over-the-counter medicines, vitamins, or herbal supplements. Especially tell your child’s healthcare provider if your child is taking aspirin, blood thinners, or other medicines to treat blood clots. Koselugo contains vitamin E, which may increase the risk of bleeding.
+        </p>
+  
+        <p class="side-effects-page--paragraph text--center color--aqua">
+          Review the treatment journey of patients who took Koselugo in a clinical study
+        </p>
+  
+        <a href="" class="text--center color--rose">See their journeys here</a>
```

Comment:
Do not leave anchors href empty, look in the figma for the proper link, if there's not, please at least add the # symbol.